### DISCUSSION TITLE
I can't make matplotlib work on my notebook (chromebook)

### DISCUSSION THREAD

**Discussion by lusidespyder** [jupyter/notebook#7297] [Q&A]
I've been having problems with the jupyter notebook on anaconda for a while now. I can't seam to get my code work when I use matplotlib, it works for any other library. It says that the kernel has died and that it will restart.

for info, i am on a chromebook and when i start anaconda i get this message (even though I reinstalled it multiple times) :

2024-03-16 19:13:32,770 - WARNING linux_scaling.get_scaling_factor_using_dbus:32
An exception occurred during fetching list of system display settings.

libGL error: MESA-LOADER: failed to open virtio_gpu: /usr/lib/dri/virtio_gpu_dri.so: cannot open shared object file: No such file or directory (search paths /usr/lib/x86_64-linux-gnu/dri:$${ORIGIN}/dri:/usr/lib/dri, suffix _dri)
libGL error: failed to load driver: virtio_gpu
libGL error: MESA-LOADER: failed to open swrast: /usr/lib/dri/swrast_dri.so: cannot open shared object file: No such file or directory (search paths /usr/lib/x86_64-linux-gnu/dri:$${ORIGIN}/dri:/usr/lib/dri, suffix _dri)
libGL error: failed to load driver: swrast
also i'm a noob at coding

I tried reinstalling anaconda multiple times
I tried updating anaconda and matplotlib
I tried using spyder but I had the same problem
I tried with miniconda as well but same

**Comment by RRosio** [comment]
Hi @lusidespyder! It looks like the issue you are facing is not specific to Notebook. There are reports of similar issues in the Anaconda repository where you might find a workaround in one of the suggestions in https://github.com/ContinuumIO/anaconda-issues/issues/12133. Please feel free to post an update with your issue and troubleshooting information, in that issue thread or repository!
